src/lib/y2firewall/dialogs/proposal.rb

```diff
@@ -55,6 +55,21 @@ def contents
             )
           )
         )
+
+        if @settings.selinux_config.configurable?
+          res.params << Frame(
```

Comment:
Curious. 

NP: I would have used a `#selinux_content` method returning `Empty()` when not configurable and called it inside the above `VBox`. Much readable to me, but maybe it does not work.

Reply:
I worry that Empty can break a bit layout as Empty is empty space on screen and no noop. That is why I dynamicaly add that frame.